src/app/constants/keyboard-keys.ts

```diff
@@ -0,0 +1,3 @@
+export const Key = {
+    Enter : 'Enter'
+};
```

Comment:
```suggestion
};

```

Please end this file with a newline